cypress/integration/spawn/volume.ts

```diff
@@ -155,14 +157,6 @@ describe("Navigating to Spawn Volume page", () => {
   // cy.contains(errorBannerCopy).should("exist");
   // });
 
-  it("Clicking on mount, selecting a host and submitting should result in a new error toast appearing.", () => {
```

Comment:
I didn't think this was a good test. The test is asserting that a valid scenario errors. But it only errors because we are unable to test it locally. It's not a good representation of what the prod behavior is so it seemed superfluous.